src/Controllers/ElementalAreaController.php

```diff
@@ -20,18 +19,17 @@
 use Exception;
 use SilverStripe\Control\HTTPRequest;
 use InvalidArgumentException;
+use SilverStripe\Admin\FormSchemaController;
 
 /**
  * Controller for "ElementalArea" - handles loading and saving of in-line edit forms in an elemental area in admin
  */
-class ElementalAreaController extends CMSMain
+class ElementalAreaController extends FormSchemaController
```

Comment:
At _worst_ it needed to be LeftAndMain before this..... I'd love to know why someone thought it had to be CMSMain back in the day.